index.js

```diff
@@ -1,10 +1,12 @@
 module.exports = {
   parser: 'babel-eslint',
   extends: ['eslint-config-airbnb'].map(require.resolve),
+  plugins: ['eslint-plugin-appjson'],
   env: {
     browser: true,
   },
   rules: {
+    'appjson/require-process-env-defined': [1, ['NODE_ENV']],
```

Comment:
I lean torward having this as a warning, but I'm open for turning it to an error.